pip/commands/__init__.py

```diff
@@ -61,18 +61,42 @@ def get_summaries(ordered=True):
         yield (name, command_class.summary)
 
 
-def get_similar_commands(name):
+def get_close_matches(word, possibilities, n=1, cutoff=0.6):
```

Comment:
This is a re-implementation that's basically copy-paste removing the last line. Why it's duplication, I don't imagine this code changing anyway. Thoughts?